tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm

```diff
@@ -145,6 +145,22 @@ sub content {
   my $headers = $header_hash->{'combined'};
   my $header_extra_descriptions = $header_hash->{'descriptions'} || {};
 
+  # Overwrite DisGeNET header description
+  if(exists $header_extra_descriptions->{'DisGeNET'}) {
```

Comment:
Do you have an example of what the description looks like?
You can also update the value like this:
$header_extra_descriptions->{'DisGeNET'} =~ s/Each value is separated.*//;

Reply:
I added an example of the description. The last part should be deleted because the output has a different structure and the values are not separated by ':'.